REVIEWING.md

```diff
@@ -0,0 +1,115 @@
+There are some exceptions to this: Updates to libraries (including Go vendor updates) that pass CI cleanly and require no code changes may be merged by a maintainer without further review.
+
+All code merged must pass CI.
+
+## What should you review?
+
+We encourage review of all PRs, even those not in the area of expertise of the reviewer.
+Even if you are not fully familiar with the code in question, you can still notice basic mistakes.
+Feel free to ask questions about how areas of the code work to help familiarize yourself during reviews.
+If you finish a review and do not feel like you adequately understood the code to approve it for merge, tag an expert in that area of the code to perform a further review.
+
+Timely PR reviews are important - contributors can become discouraged if a PR is neglected by maintainers.
+All Maintainers and Reviewers should try to review new pull requests in their repositories once a day to ensure this.
+When you review a PR with failing tests, please check to see if those tests failed due to known flakes.
+If so, please restart the failed tests.
+Many repositories in the Podman project can only have their tests restarted by project members, not the submitter, so regular attention to test failures by reviewers is important.
+
+## Things to Check
+
+### Breaking Changes
+
+The Podman project aims to present a stable API for its users.
+Breaking changes to the project's Command Line Interfaces or public APIs (the Podman REST API and its associated bindings) must only be made in approved breaking change releases - Podman 6.0, 7.0, etc.
+Individual repositories should identify what parts of the repository are considered to be their stable API.
+Breaking changes can include renaming an option without retaining the original name as an alias, removing an option entirely, or changing how an option works in a way that does not ensure backwards compatibility.
+
+Periods when it is acceptable to merge breaking changes will be widely announced.
+If it is not one of those periods, reviewers should be on the lookout for breaking changes.
+PRs with breaking changes should not be merged.
+Please guide the contributor in how to make the change in a non-breaking fashion.
+If this is not possible, deferring the PR to the next breaking change window or closing it entirely is appropriate.
+
+### Commit Messages and Hygiene
+
+Good commit messages are essential for understanding why a change was made in the future.
+Reviewers should check each commit of a PR to ensure that it has an appropriate commit message which fully and accurately explains the change and why it is being made.
+This should remain true after changes to the PR due to code review, so please re-review commit messages before merging code, to ensure they are still accurate.
+Pull requests fixing a specific issue must include a `Fixes: #xxxxx` line in the commit message.
```

Comment:
in the case of Jira, I think @Luap99 recommends the plain link instead of a GH reference.  @Luap99 what say you?

Reply:
Sure but IMO this document should not need duplicate what is written in CONTRIBUTING.md, i.e.
https://github.com/containers/podman/blob/main/CONTRIBUTING.md#describe-your-changes-in-commit-messages

Overall I think most part should be ensure CONTRIBUTING.md is followed, this here doc here should then just help is small review guidance so people don't forget to double check these things but there should be no need to go into the full details here again